packages/scxa-cell-type-wheel/cypress/component/CellTypeWheel.cy.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import CellTypeWheel from '../../src/CellTypeWheel'
 
 import cellTypeWheelData from '../fixtures/lung-cancer.json'
+import bigCellTypeWheelData from '../fixtures/cancer.json'
```

Comment:
Now that we have two payloads, we can give them proper names such as `lungCancerCellTypeWheelData` or `cancerCellTypeWheelData`. It may be useful to comment in the test, or here in the `import`, the specific browser and OS where this unusual situation happens, and a link to the GitHub issue in Highcharts.